pkg/models/images.go

```diff
@@ -21,6 +21,7 @@ type Image struct {
 	Commit       *Commit    `json:"Commit"`
 	InstallerID  *uint      `json:"InstallerID"`
 	Installer    *Installer `json:"Installer"`
+	ParentId     int        `gorm:"foreignKey:Image"`
```

Comment:
```suggestion
	ParentId     *uint        `gorm:"foreignKey:Image"`
```

I think we need to make this nullable and make sure it's unsigned